src/search/SearchForm.tsx

```diff
@@ -100,6 +128,12 @@ const SearchForm = ({ onSearch, formValues }: Props): JSX.Element | null => {
     Object.keys(formValues).forEach((p) => setValue(p, formValues[p]));
   }, [formValues, setValue]);
 
+  useEffect(() => {
+    if (showMoreFilters) {
+      document.getElementById('participantCountFilter-toggle-button')?.focus();
```

Comment:
No biggie, but could this be resolved with a `ref`?

Reply:
yeah, I tried to figure that out, but was not able to because select does not support ref.